src/python/src/grpc/_adapter/_c/module.c

```diff
@@ -0,0 +1,26 @@
+#include <stdlib.h>
+
+#include <grpc/grpc.h>
+
+#include "grpc/_adapter/_c/types.h"
+
+static PyMethodDef c_methods[] = {
```

Comment:
It has to be defined _somewhere_ at some point (no type inference in C). Would you prefer it to be inside `init_c`?